src/i18n/resources/en/fra.js

```diff
@@ -7,6 +7,16 @@ module.exports = {
   forestArea100HaYear: 'Forest area (1000 ha)',
 
   // tables
+  areaAffectedByFire: {
+    areaUnitLabel2025: 'Area affected (1000 ha)',
+  },
+
+  disturbances: {
+    disturbances2025: 'Forest damage',
+    categoryHeader: 'Predominant cause',
+    areaUnitLabel2025: 'Forest area affected (1000 ha)',
```

Comment:
can these be predominantCause and forestAreaAffected ? does it match 2025 data export unit label?